src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -303,11 +368,24 @@ public String getApiMethodName(Method method, VisibilityConfig visibility) {
     return getApiMethodName(Name.upperCamel(method.getSimpleName()), visibility);
   }
 
+  /** The name of the surface method which can call the given API method. */
+  public String getApiMethodName(
+      com.google.api.codegen.discovery.Method method, VisibilityConfig visibility) {
+    return getApiMethodName(Name.anyCamel(discoveryMethodNamePieces(method)), visibility);
+  }
+
   /** The name of the async surface method which can call the given API method. */
   public String getAsyncApiMethodName(Method method, VisibilityConfig visibility) {
```

Comment:
"Method: The Abstraction", starring Java Interface, is coming soon to a PR near you!

The pairs of methods are a stopgap and will be ELIMINATED.